stdlib/builtins.pyi

```diff
@@ -968,7 +968,8 @@ class tuple(Sequence[_T_co]):
     if sys.version_info >= (3, 9):
         def __class_getitem__(cls, item: Any, /) -> GenericAlias: ...
 
-# Doesn't exist at runtime, but deleting this breaks mypy. See #2999
+# Doesn't exist at runtime, but deleting this breaks mypy. See https://github.com/python/typeshed/issues/7580
+# and https://github.com/python/mypy/issues/8240.
```

Comment:
```suggestion
# and https://github.com/python/mypy/issues/8240
```

or the link doesn't work when you click on it from the GitHub UI.

Also, pyright also uses these classes: https://github.com/python/mypy/issues/8240#issuecomment-1086372894